roles/nuage_master/tasks/firewall.yml

```diff
@@ -0,0 +1,40 @@
+---
+- when: os_firewall_enabled | bool and not os_firewall_use_firewalld | bool
+  block:
+  - name: Add iptables allow rules
+    os_firewall_manage_iptables:
+      name: "{{ item.service }}"
+      action: add
+      protocol: "{{ item.port.split('/')[1] }}"
```

Comment:
nit: Calling on it once here ... I think it would be cleaner to separate protocol and port in the defaults. This would make it easier on the eyes. Anywhere that needs ``proto/port`` would read better as ``{{ $ITEM_protocol }}/{{ $ITEM_PORT }}`` as well.